meta-xt-prod-cockpit-rcar-control/recipes-guests/domd/files/domd-h3ulcb-4x2g-kf.cfg

```diff
@@ -0,0 +1,336 @@
+# =====================================================================
+# DomD guest configuration
+# =====================================================================
+seclabel='system_u:system_r:domD_t'
+# Guest name
+name = "DomD"
+# Kernel image to boot
+kernel = "/usr/lib/xen/boot/linux-domd"
+device_tree = "/usr/lib/xen/boot/domd.dtb"
+# Kernel command line options
+# Uncomment this when using nfs as a boot device
+#extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/domd,vers=3 ip=dhcp rw rootwait console=hvc0 pvrsrvkm.DriverMode=0 xt_page_pool=67108864 xt_cma=100663296"
```

Comment:
Could you please remove lines 11 and 12? NFS boot is enabled in other way now.

This leftover from old prod-devel adds confusion.